packages/integrations/cloudflare/src/index.ts

```diff
@@ -233,14 +235,39 @@ export default function createIntegration(args?: Options): AstroIntegration {
 				// cloudflare to handle static files and support _redirects configuration
 				// (without calling the function)
 				if (!routesExists) {
+					const functionEndpoints = routes
+						.filter((route) => ['endpoint', 'page'].includes(route.type) && !route.prerender)
```

Comment:
You can move `['endpoint', 'page']` inside a top-level `const`, so we don't create the same array at each iteration. Also, could you leave a comment that explains why we have this filter?